benchmark_duckdb.py

```diff
@@ -87,19 +96,31 @@ def run_benchmark(con, trec_output_file_path, metric):
                     evaluation_metric = 'array_inner_product'
 
                 sql_query = f"SELECT id, {evaluation_metric}(vector, ?::FLOAT[{len(vector)}]) as score FROM documents ORDER BY score DESC LIMIT ?"
+                # time the execution
+                start_time = time.time()
                 results = con.execute(sql_query, (vector, K)).fetchall()
+                end_time = time.time()
+                # aggregate time
+                total_time += end_time - start_time
```

Comment:
let's grab every time in the query set and we can sum them up later, but also get stats on mean/variance/min/max